### DISCUSSION TITLE
How send review and feedback request separately  ?

### DISCUSSION THREAD

**Discussion by Lokeesh-27** [amzn/selling-partner-api-models#3711] [General]
Using the SP API - solicitation API, we can send review and feedback request template together. However, I need to send them **separately**. how to send it separately ?

Some third-party tools (SAAS) allow you to send review and feedback requests separately, with custom templates for each. However, the Selling Partner API currently requires sending a single template for both review and feedback requests.
I need to know is there any other method to achieve this ?

**Comment by StrangeTim** [comment]
According to the [use case](https://developer-docs.amazon.com/sp-api/docs/solicitations-api-use-case-guide) guide and the [model methods](https://developer-docs.amazon.com/sp-api/docs/solicitations-api-v1-reference) available, doing these operations separately is not supported yet. This may be a good thing to put in the [Ideas](https://github.com/amzn/selling-partner-api-docs/discussions/categories/ideas) channel.

**Reply by Lokeesh-27** [reply]
Some third-party tools (SAAS) allow you to send review and feedback requests separately, with custom templates for each. However, the Selling Partner API currently requires sending a single template for both review and feedback requests.
I need to know is there any other method to achieve this ?
